launch/launch.go

```diff
@@ -54,28 +54,32 @@ type Option struct {
 	ArtifactsPath string
 	Memory        string
 	SrcPath       string
+	OptionEnv     EnvVar
 }
 
 const (
 	defaultArtDir = "/sd/workspace/artifacts"
 )
 
-func mergeEnv(env, userEnv envVar) []envVar {
+func mergeEnv(env, userEnv, optionEnv EnvVar) []EnvVar {
```

Comment:
[nits]
Is it better to change the name of `userEnv`?  It takes time to me to understand what is userEnv

Reply:
Both envs are looks "user's".  How about using `jobEnv` and `optionEnv`?

Reply:
Thx.
I fixed it.